addons/sale/models/sale.py

```diff
@@ -837,8 +818,30 @@ def _prepare_add_missing_fields(self, values):
                     res[field] = line._fields[field].convert_to_write(line[field], line)
         return res
 
+    def _force_line_type_vals(self, vals, is_create=False):
+        """ Depending on the line_type, there are some values that we want or don't want to be set.
+        """
+        line_type = vals.get('line_type')
+        if not line_type:
+            if is_create:
+                line_type = self.default_get(['line_type'])['line_type']
+            else:
+                return vals
+        if line_type != 'line_section':
+            vals['line_pagebreak'] = False
```

Comment:
Same here, line type should be required and you could pop the values instead.

Reply:
The IF here is not because line-type is not required, but because in case of an update (write), the line type will not be present in vals because it shouldn't/won't be changed.

Maybe I should "compute" the line_type separately in the create/write and make this method static with the line_type given as parameter to get the proper default value?